calyx-py/test/correctness/queues/strict_and_rr_queues/gen_strict_or_rr.py

```diff
@@ -33,7 +34,7 @@ def invoke_subqueue(queue_cell, cmd, value, ans, err) -> cb.invoke:
 
 
 def insert_strict_pifo(
-    prog, name, fifos, boundaries, numflows, order, queue_len_factor=QUEUE_LEN_FACTOR
+    prog, name, fifos, boundaries, numflows, order, round_robin, queue_len_factor=QUEUE_LEN_FACTOR
 ):
     """Inserts the component `pifo` into the program."""
```

Comment:
Docstring could use a little more info, especially about how the same method can be used both ways!